clusters/kind-cluster/base/namespaces.yaml

```diff
@@ -4,4 +4,21 @@ metadata:
   labels:
     app.kubernetes.io/instance: dscp
     app.kubernetes.io/version: latest
-  name: dscp
+  name: alice
+---
```

Comment:
aren't you missing `...`  to indicate the end of a document?

Reply:
Nope.